translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/ProjectManager/tasks/resync/ProjectListResyncTask.java

```diff
@@ -34,15 +38,16 @@
 
     Context mCtx;
     FragmentManager mFragmentManager;
+    ProjectDatabaseHelper mDb;
 
-    public ProjectListResyncTask(int taskId, Context ctx, FragmentManager fm) {
+    public ProjectListResyncTask(int taskId, Context ctx, FragmentManager fm, ProjectDatabaseHelper db) {
```

Comment:
if you inject the chunkpluginloader, you won't need the context object dependency here